test_main.py

```
@@ -238,8 +238,86 @@ def test_export_unlabeled_nodes(self, falkordb_connection, temp_dir):
-    def test_export_connection_error(self):
+    def test_export_connection_error(self, temp_dir):
         """Test behavior when FalkorDB connection fails."""
+        # Test with localhost on a port that's definitely not listening
+        # This should cause a connection refused error
         with pytest.raises(Exception):
-            # Try to connect to non-existent server
-            export_graph("test_graph", "nonexistent_host", 9999)
\ No newline at end of file
+            export_graph("test_graph", "localhost", 1)
+    
+    def test_export_with_auth_params(self, test_graph, temp_dir):
```

Ruff (0.14.2)

Unused method argument: `temp_dir`

(ARG002)